scenarios/aws/ec2/vmargs.go

```diff
@@ -81,3 +83,10 @@ func WithInstanceProfile(instanceProfile string) VMOption {
 		return nil
 	}
 }
+
```

Comment:
I think it makes sense to add some documentation for this function as the name of the field in ec2.MetadataOptions is not obvious.

Reply:
I agree something like `WithEnableIMDS2` would be clearer